### PR TITLE
Document HAProxy installation and ConductR HAProxy setup for Mesos

### PR DESCRIPTION
**Pull request by fsat**

### CODE REVIEW

File: src/main/play-doc/operation/Install.md [hunk 1]
```diff
@@ -729,6 +784,31 @@ SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment va
...
+* Download the file.
+
+Once the file is downloaded, copy this file to the bastion host, e.g. the `/tmp` directory.
+
+SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment variable with the host or ip address of one of the ConductR Core that's currently running.
+
+Load and run the ConductR-HAProxy bundle as follows. Scale ConductR-HAProxy so that ConductR-HAProxy is running on every proxy node in the cluster. Assuming we have 3 public slave nodes where the proxy is expected to be running, we need scale up the ConductR-HAProxy to 3 instances.
+
+```bash
+conduct load file:/tmp/conductr-haproxy-{version}-{digest}.zip
```
Comment (fsat) [inline src/main/play-doc/operation/Install.md:805]: @huntc (cc @edwardcallahan ): are we going to publish our bundles (i.e. HAProxy, ES, and Visualizer) to bintray as part of 2.0 release?

If we do, the step to obtain the ConductR HAProxy bundle file can be removed and the install steps can be cut down significantly.

**Comment by fsat** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;aLSayumh&#x2F;301-document-haproxy-installation-and-setup-to-support-running-conductr-haproxy-on-mesos-mode"> Document HAProxy installation and setup to support running ConductR HAProxy on Mesos mode</a>

**Comment by fsat** [comment]
Asked the Fastdata team if there's a way to copy a file out of the running task using DCOS CLI. This is to hopefully cut down the number steps to obtain ConductR HAProxy bundle.

**Comment by fsat** [comment]
Marked as wip for now - I have some questions pending.

File: src/main/play-doc/operation/Install.md [hunk 2]
```diff
@@ -729,6 +784,31 @@ SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment va
...
+ConductR-HAProxy bundle will expose services provided by the bundle so it can be accessed from outside cluster. ConductR-HAProxy will ensure the HAProxy configuration is kept update based on the bundles which are running.
+
+Obtain the ConductR HAProxy bundle from the ConductR installation.
+
+* Open the Mesos Web Console
+* Locate the ConductR Core task running in the list of Active Tasks. The task is called `conductr-{version}` or something similar.
+* Click the Sandbox link to view the filesystem of the deployed ConductR Core.
+* Navigate to the directory `conductr-{version}-{digest}/extra`.
+* Locate the ConductR HAProxy bundle in a file called `conductr-haproxy-v2-{digest}.zip`.
+* Download the file.
```
Comment (fsat) [inline src/main/play-doc/operation/Install.md]: Asking the Fastdata team if this can be achieved using DCOS CLI.

Comment (huntc) [inline src/main/play-doc/operation/Install.md]: Why not just download the ConductR distribution the bastin host and then distribute the file from there?

File: src/main/play-doc/operation/Install.md [hunk 3]
```diff
@@ -729,6 +784,31 @@ SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment va
...
+
+Once the file is downloaded, copy this file to the bastion host, e.g. the `/tmp` directory.
+
+SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment variable with the host or ip address of one of the ConductR Core that's currently running.
+
+Load and run the ConductR-HAProxy bundle as follows. Scale ConductR-HAProxy so that ConductR-HAProxy is running on every proxy node in the cluster. Assuming we have 3 public slave nodes where the proxy is expected to be running, we need scale up the ConductR-HAProxy to 3 instances.
+
```
Comment (huntc) [inline src/main/play-doc/operation/Install.md:803]: You don't need to scale to the number of public slaves. How about we recommend just 3 as a starting point?

File: src/main/play-doc/operation/Install.md [hunk 4]
```diff
@@ -749,23 +829,4 @@ NAME           HOST       USER  STATE  ID
...
-The services exposed by the bundles are not easily accessible from outside of the ConductR cluster until the consolidation work is completed.
-
-However, the services is still accessible once the host address and the port where the services is running is known.
-
-Use the Control API to [query bundle state](ControlAPI#Query-bundle-state). Each bundle that has been started will have `bundleExecutions` field populated with the respective `host` where the bundle is running and `bindPort` where the service is exposed.
-
-### Accessing services from within cluster
-
-The services exposed by the bundles can be resolved by [performing a service lookup](ResolvingServices) against any of the ConductR node.
+The `compatibilityVersion` further explained in the to [bundle configuration](BundleConfiguration).
```
Comment (huntc) [inline src/main/play-doc/operation/Install.md]: "is explained"

File: src/main/play-doc/operation/Install.md (hunk 1 above)
Comment (huntc) [inline src/main/play-doc/operation/Install.md:805]: Don't know yet

File: src/main/play-doc/operation/Install.md (hunk 2 above)
Comment (fsat) [inline src/main/play-doc/operation/Install.md]: Are we going to make the ConductR Core `.tgz` available on download page? If so, then I'll update the docs to get the users to download this and extract on the bastion host.

Comment (huntc) [inline src/main/play-doc/operation/Install.md]: There are no plans to change things here i.e. the ConductR production download remains available from where it is today. This should be no different to regular EC2/Linux installations.

Comment (edwardcallahan) [inline src/main/play-doc/operation/Install.md]: FTR, yes, the universal tgz package will be available on the download page along with the deb and rpm distros.

File: src/main/play-doc/operation/Install.md [hunk 5]
```diff
@@ -729,6 +784,28 @@ SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment va
...
+
+## Loading and Running ConductR-HAProxy Bundle
+
+ConductR-HAProxy bundle will expose services provided by the bundle so it can be accessed from outside cluster. ConductR-HAProxy will ensure the HAProxy configuration is kept update based on the bundles which are running.
+
+First we need to obtain the `.tgz` ConductR installations package.
+
+> In order to obtain the installations of ConductR then please [contact our sales department](https://www.lightbend.com/company/contact). To evaluate ConductR in general then [please visit our product page](http://www.lightbend.com/products/conductr) which provides instructions on getting started. Otherwise if you are looking to use ConductR for free from a development perspective then please [head over to our developer section](DevQuickStart).
+
+Once the ConductR `.tgz` is obtained, extract ConductR HAProxy bundle from the package. The ConductR HAProxy bundle can be found within the ConductR package under  `conductr-{version}/extra/conductr-haproxy-{version}-{digest}.zip`.
```
Comment (fsat) [inline src/main/play-doc/operation/Install.md:796]: @huntc and @edwardcallahan - obtaining ConductR HAProxy bundle is now done through downloading Conductr `.tgz` from download page.

File: src/main/play-doc/operation/Install.md [hunk 6]
```diff
@@ -749,23 +826,4 @@ NAME           HOST       USER  STATE  ID
...
-The services exposed by the bundles are not easily accessible from outside of the ConductR cluster until the consolidation work is completed.
-
-However, the services is still accessible once the host address and the port where the services is running is known.
-
-Use the Control API to [query bundle state](ControlAPI#Query-bundle-state). Each bundle that has been started will have `bundleExecutions` field populated with the respective `host` where the bundle is running and `bindPort` where the service is exposed.
-
-### Accessing services from within cluster
-
-The services exposed by the bundles can be resolved by [performing a service lookup](ResolvingServices) against any of the ConductR node.
+The `compatibilityVersion` is explained in the to [bundle configuration](BundleConfiguration).
```
Comment (fsat) [inline src/main/play-doc/operation/Install.md:829]: @huntc - update to "is explained"

File: src/main/play-doc/operation/Install.md [hunk 7]
```diff
@@ -729,6 +784,28 @@ SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment va
...
+
+> In order to obtain the installations of ConductR then please [contact our sales department](https://www.lightbend.com/company/contact). To evaluate ConductR in general then [please visit our product page](http://www.lightbend.com/products/conductr) which provides instructions on getting started. Otherwise if you are looking to use ConductR for free from a development perspective then please [head over to our developer section](DevQuickStart).
+
+Once the ConductR `.tgz` is obtained, extract ConductR HAProxy bundle from the package. The ConductR HAProxy bundle can be found within the ConductR package under  `conductr-{version}/extra/conductr-haproxy-{version}-{digest}.zip`.
+
+Upload the ConductR HAProxy bundle to the bastion host, e.g. the `/tmp` directory.
+
+SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment variable with the host or ip address of one of the ConductR Core that's currently running.
+
+Load and run the ConductR-HAProxy bundle as follows. Scale up the ConductR-HAProxy to 3 instances as a starting point.
```
Comment (fsat) [inline src/main/play-doc/operation/Install.md:802]: @huntc - scaling proxy to 3 instances as a starting point.

**Pull request merged**